### PR TITLE
Create from description command

### PR DESCRIPTION
**Pull request by maxim-saplin**
Thanks for the nice extension! I've added a new command which I missed while using the extension - ability to create code via a prompt and have it inserted at the current cursor location. Please not that my version also added the concept of context and puts surrounding code to the prompt.

### CODE REVIEW

File: src/tools/create/code_from_description.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        }, async (progress) => {
+            let progressPercentage = 0;
+            let prevProgressPercentage = 0;
+            const progressInterval = setInterval(() => {
+                prevProgressPercentage = progressPercentage;
+                progressPercentage = (progressPercentage + 10) % 100;
+                const increment = progressPercentage - prevProgressPercentage;
+                progress.report({ increment });
+            }, 200);
+            let prompt = `You're an expert Flutter developer. Follow the instructions carefully and to the letter.\n\n`;
```
Comment (samyakkkk) [inline src/tools/create/code_from_description.ts:33]:
```suggestion
            let prompt = `You're an expert Flutter/Dart coding assistant. Follow the user instructions carefully and to the letter.\n\n`;
```
Keeping the identity consistent across all platforms.

File: package.json [hunk 2]
```diff
@@ -77,6 +77,15 @@
         "title": "API Repository from Postman JSON",
         "when": "false"
       },
+      {
+        "command": "fluttergpt.createCodeFromDescription",
+        "title": "FlutterGPT Create: From Description"
+      },
+      {
+        "command": "fluttergpt.createCodeFromDescription.menu",
```
Comment (samyakkkk) [inline package.json:85]: Register this in `fluttergpt.createMenu` as well so the command is also accessible via right-click menu under FlutterGPT: Create section.

**Review by samyakkkk — CHANGES_REQUESTED** [review]
hey @maxim-saplin! thanks for bringing in your ideas on table. 

I tried it out and looks good :) the existing `codeFromBlueprint` function we have can be tuned to do the same but I think your code to add the context along with the instructions brings more value to this case. 

Would love to merge and push this in the next release. Just left some minor change requests. Please address them. 
 
Thanks!

**Comment by maxim-saplin** [comment]
Thanks for the quick reply and review. I hope I didn't miss anything in the updates I just did

**Comment by samyakkkk** [comment]
Thanks @maxim-saplin, merging this. It will go in the next release!

**Pull request merged**